src/planner_test.cc

```diff
@@ -17,6 +16,69 @@
 // TODO: This should work being set to 1
 #define SPEED_STEP_FACTOR 4
 
+// Helper function for implementing ASSERT_NEAR.
+testing::AssertionResult DoubleNearAbsRelPredFormat(const char* expr1,
```

Comment:
this looks more complicated than I'd expect, in particular since below in your test, you really only test relative errors, and have the absolute error set to zero.

So I'd reduce it: only write more code once it is needed.

Reply:
(solved)